tools/barista/README.md

```diff
@@ -27,12 +27,25 @@ node dist/tools/barista/main.js
 
 ### Environment variables
 
-- `PUBLIC_BUILD` (optional): whether the build should only output public pages
+- `PUBLIC_BUILD` (optional): whether the build should only output public pages;
+  must be explicitly set to `false` to build the internal Barista version
 - `STRAPI_ENDPOINT`: the CMS endpoint
 - `INTERNAL_LINKS`: (parts of) internal links that should be removed from the
   content on public build; separated by `,` (is transformed to the following
   selector or a list of selectors: `a[href*="<internal-link>"]`)
 
+### Configurable directories (optional)
```

Comment:
I think this information is now outdated ;) please adapt it to the environment files ;)